vl-proza-message.src.js

```diff
@@ -208,15 +226,38 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
 
     __processKeydownEvent(e) {
         if (this.__isEscapeKey(e)) {
-            this.__undoAllWysiwygChanges();
-            this.__stopWysiwyg();
+        	this.__cancel();
         }
         if (this.__isEnterKey(e) && !this.__isShiftKey(e)) {
-            this.__undoWysiwygChange();
-            this.__stopWysiwyg();
+            this.__undoWysiwygChange(); //enter verwijderen
+            this.__save();
         }
     }
 
+    __processBlurEvent() {
+    	this.__save();
+    }
+
+    __save() {
+    	ProzaRestClient.saveMessage(this._domain, this._code, this._activeWysiwygEditor.getContent()).then(message => {
+    		VlProzaMessage._putInCache(this._domain, this._code, Promise.resolve(message));
+    		this.__stopWysiwyg();
+    	}).catch((error) => {
+    		const alert = this._template(`
+	    		<vl-alert type="error" icon="alert-triangle" title="Technische storing" closable>
+	              <p>Uw wijziging kon niet bewaard worden. Probeer het later opnieuw of neem contact op met de helpdesk als het probleem zich blijft voordoen.</p>
+	            </vl-alert>
```

Comment:
Misschien moeten we ons hier nu geen zorgen over maken, maar momenteel ondersteunen we hier geen i18n. We kunnen dat uitbreiden wanneer relevant, alleen Proza beheerders zien de eventuele foutmelding en die kunnen vermoedelijk altijd Nederlands.